website/content/api-docs/secret/pki.mdx

```diff
@@ -3101,6 +3132,76 @@ $ curl \
 }
 ```
 
+### Read Cluster Configuration
+
+This endpoint fetches the cluster-local configuration.
+
+Presently the only cluster-local config option is `path`, which sets the URL
+to this mount on a particular performance replication cluster. This is useful
+for AIA URL templating.
```

Comment:
Should we call the variable out explicitly, here and possibly below?

`This is useful for populating '{{cluster_path}}' within AIA URL templating`

Reply:
Added, also added a notice that we may use this for other uses in the future.